src/assets/sass/scratch/_toolbox.scss

```diff
@@ -34,82 +32,118 @@ $category-colours: (
         overflow: auto;
     }
-    &__row {
-        cursor: pointer;
-        border-top: 1px solid $lightgrey;
-    }
-    &__header, &__label {
-        font-size: 1em;
-        padding: 0.6em 1em;
-    }
-    &__title {
-        margin-right: 0.6em;
-    }
-    &__arrow {
+    &__cat-return {
         cursor: pointer;
-        position: absolute;
-        top: 0.6em;
-        right: 0.6em;
-        transform: rotate(90deg);
-        transition: transform 0.25s ease;
+        color: $white;
+        text-align: center;
+
+        #{$toolbox}__label {
+            margin-left: -($cat-text-spacing * 2);
+        }
     }
-    &__item {
-        position: relative;
+    &__cat-text {
+        align-self: center;
+        margin: $cat-text-spacing;
+        width: 100%;
     }
-    &__color {
-        position: absolute;
-        left: 0;
-        width: 4px;
-        height: 100%;
+    &__description {
+        font-size: 1.3em;
     }
-    &--collapsed {
-        #{$toolbox}__header, #{$toolbox}__item {
-            padding: 0.6em;
-        }
-        #{$toolbox}__color {
-            position: relative;
-            width: 20px;
-            height: 20px;
-            border-radius: 50%;
-        }
-        #{$toolbox}__title, #{$toolbox}__label {
-            display: none;
-        }
-        #{$toolbox}__arrow {
-            position: relative;
-            transform: rotate(270deg);
-            margin-top: -2px;
-            top: 0;
-            right: -2px;
-        }
-        #{$toolbox}__row {
-            border-top: none;
+    &__icon {
+        display: flex;
+        align-self: center;
+        margin-right: $cat-text-spacing;
+
+        > svg {
+            height: 25px;
+            width: 25px;
         }
     }
-    @each $category, $category-colour in $category-colours {
-        &__category--#{$category}#{&}__category--selected {
-            background-color: transparentize($category-colour, 0.9);
+    &__label {
+        font-weight: bold;
+        font-size: 1.6em;
+    }
+    &__row {
+        cursor: pointer;
+        border-top: 1px solid $lightgrey;
+
+        &:last-of-type {
+            border-bottom: 1px solid $lightgrey;
         }
-        &__category--#{$category} {
-            &:hover {
-                background-color: transparentize($category-colour, 0.95);
-            }
-            & > #{$toolbox}__color {
-                background-color: $category-colour;
+
+        @each $category, $colour in $category-colours {
+            #{$toolbox}__cat {
+                &--#{$category} {
+                    &:hover:not(#{$toolbox}__cat--selected) {
+                        background-image: linear-gradient(to right, $white 25%, transparentize($colour, 0.8) 100%);
+                    }
+                }
+                &--#{$category}#{$toolbox}__cat--selected {
+                    background-color: transparentize($colour, 0.9);
+                    transition: background-color 0.25s ease;
```

Comment:
@aaron-binary and also pls add mixin and browser support for transition.

Reply:
I've added the mixin as part of #14